compiler/hermes/CMakeLists.txt

```diff
@@ -3,7 +3,11 @@ file(GLOB_RECURSE TESTS "src/*.test.cpp")
 list(REMOVE_ITEM SOURCES ${TESTS})
 
 add_library(hermes STATIC ${SOURCES})
-set_target_properties(hermes PROPERTIES POSITION_INDEPENDENT_CODE ON)
+
```

Comment:
In other PR's you don't adds empty line before `if` statement. Redundant?

Reply:
Removed =)